packages/adp-tooling/src/base/prompt.ts

```diff
@@ -28,6 +26,7 @@ export async function promptGeneratorInput({
                 { title: flexLayer.CUSTOMER_BASE, value: flexLayer.CUSTOMER_BASE },
                 { title: flexLayer.VENDOR, value: flexLayer.VENDOR }
             ],
+            initial: flexLayer.CUSTOMER_BASE,
```

Comment:
Adding initial will not work, it will throw an exception, please see here https://github.com/SAP/open-ux-tools/pull/1105#issuecomment-1641779652. That is why I removed it in this commit https://github.com/SAP/open-ux-tools/pull/1105/commits/79a2b56bdc0646940dc775d21a27e210c3f82b74

Reply:
That's interesting. I need to have a closer look.

Reply:
Checked it, for a selection the initial value needs to be the index of the choices. Default is `0` which is already `CUSTOMER_BASE` so I can just remove it again.
@zdravko-georgiev thanks for the catch!

Reply:
hmm ... but if I revert it then the unit tests fail. Also if I see `0` explicitly.

Reply:
Yes, I remember now. It was tricky to unit test the selection of the initial value...

Reply:
https://github.com/SAP/open-ux-tools/pull/1143/commits/1f96c8e5ef35e564318951a7de57e0a52d43372a